elements/rh-cta/rh-cta.css

```diff
@@ -4,19 +4,20 @@
   z-index: 0;
   align-items: center;
   max-width: max-content;
-  cursor: pointer;
 }
 
 ::slotted(*) {
   vertical-align: middle;
-  white-space: normal;
-  display: inline;
+  white-space: break-spaces;
+  display: inline !important;
   color: inherit !important;
   font-family: inherit !important;
   font-size: inherit !important;
   font-weight: inherit !important;
   line-height: inherit !important;
   text-decoration: var(--rh-cta-text-decoration);
+  position: relative;
+  z-index: 2;
```

Comment:
Wonder if this could need an escape hatch?  `var(--rh-cta-zindex, 2)`. Assuming this is only ever in its own context z-index via the shadowroot.

Reply:
I don't think we do. The slotted content should be a single child with no lightDOM siblings to be affected by stacking order. That z-index sets it relative to the slot's siblings in the template.